# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/test/gtest/parent/moz.build
# Commit: 6d1f25188b13
# Full Hash: 6d1f25188b13f5c88f36bcb32caa2b74216f81ab
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 2
# Description:
#   Bug 1786465 - Add support for asynchronous closing of FileSystemDataManager; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155787
# ==============================================================================

diff -r 0256861a7ac5 -r 6d1f25188b13 dom/fs/test/gtest/parent/moz.build
--- a/dom/fs/test/gtest/parent/moz.build	Wed Aug 31 09:34:57 2022 +0000
+++ b/dom/fs/test/gtest/parent/moz.build	Wed Aug 31 09:34:58 2022 +0000
@@ -4,6 +4,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 
+TEST_DIRS += ["datamodel"]
+
 UNIFIED_SOURCES = [
     "TestFileSystemHashSource.cpp",
 ]
